pyannote/database/loader.py

```diff
@@ -187,6 +187,47 @@ def __call__(self, file: ProtocolFile) -> Timeline:
         return self.loaded_[uri]
 
 
+class LabLoader:
```

Comment:
For consistency with other loaders:

```suggestion
class LABLoader:
```